src/Aspire.Hosting.Azure.PostgreSQL/AzurePostgresExtensions.cs

```diff
@@ -52,12 +52,11 @@ public static class AzurePostgresExtensions
             var administratorLogin = new Parameter("administratorLogin");
             var administratorLoginPassword = new Parameter("administratorLoginPassword", isSecure: true);
 
-            var postgres = new PostgreSqlFlexibleServer(construct, administratorLogin, administratorLoginPassword, name: construct.Resource.Name);
+            var postgres = new PostgreSqlFlexibleServer(construct, administratorLogin, administratorLoginPassword, name: construct.Resource.Name, storageSizeInGB: 32);
             postgres.AssignProperty(x => x.Sku.Name, "'Standard_B1ms'");
             postgres.AssignProperty(x => x.Sku.Tier, "'Burstable'");
             postgres.AssignProperty(x => x.Version, "'16'");
             postgres.AssignProperty(x => x.HighAvailability.Mode, "'Disabled'");
-            postgres.AssignProperty(x => x.Storage.StorageSizeInGB, "32");
```

Comment:
Does this mean this value isn't settable via AssignProperty?

Reply:
Actually, I was mistaken - overriding the property should work now because the underlying nested instances won't be null.